pkgs/development/compilers/llvm/14/lldb/default.nix

```diff
@@ -141,5 +144,6 @@ stdenv.mkDerivation (rec {
 
   meta = llvm_meta // {
     description = "man pages for LLDB ${version}";
+    broken = true; # TODO: sort out python installation
```

Comment:
i'd just remove the `lib` output because the python dir has a relative symlink to a file in lib

```
readlink -f /nix/store/x79rab8skjafp1cn6l1nmx0iwlkpwqfw-lldb-14.0.0/lib/python3.9/site-packages/lldb/_lldb.cpython-39-x86_64-linux-gnu.so
/nix/store/x79rab8skjafp1cn6l1nmx0iwlkpwqfw-lldb-14.0.0/lib/liblldb.so
```

Reply:
Thanks for helping sort this out!

Hmm, actually yeah maybe that's the way to go.  At least in terms of impact on closure, I don't think there are (m)any users of lldb.lib so dropping that shouldn't have impact there.  Not sure if there are other reasons to motivate it (as a user, not having libraries in my profile is kinda nice but shrug).

I hacked together some exploratory commits that seem to fix things, but I don't have enough time to sit down and vet them (and apologies for slightly messy commit history): https://github.com/dtzWill/nixpkgs/commits/experimental/lldb14-py .

One fix we probably want is: f3b1fa8d90c16f4e80d56dae01c2ba1c6a0d07bc to fix the installCheck :).

https://github.com/dtzWill/nixpkgs/commit/4d722e421de2816fd75ec6d5090200ea29b722b2 seems to fix the installation (won't apply and commit message might not make sense as-is because I previously added more to our lldb `gnu-install-dirs.patch`, but it's trivial to resolve that if we like it).

Actually, I'll just do that now in case it helps.  Sorry I'll see if I can revisit this soon but might not for a few days or more.

Here's just those two commits, which seems to fix LLDB14.  The patch is somewhat inspired by one of the fixes from LLDB13 we dropped-- that fix still applies but causes python to be installed to /both/ outputs somehow! (identical files, not symlinked).  Anyway here it is: https://github.com/dtzWill/nixpkgs/tree/fix/lldb-py . Presumably same works for the git variant, but I haven't check.

Reply:
Okay let's work on how to resolve that separately.

For now keep the installCheckPhase's in our broken LLDB's as-is, will sort how we want to address this (collapse the outputs, patch the build system, mark broken, ...) in a dedicated PR.  Thanks!